package.json

```diff
@@ -4,8 +4,8 @@
     "postinstall": "node script/get-quasar.js",
     "clean": "rimraf output && rimraf .pulp-cache && rimraf test/tmp",
     "build": "pulp build -- --censor-lib --strict",
-    "property-tests": "pulp test --test-path test/src --main Test.Property.Main",
-    "test": "pulp test --test-path test/src --main Test.Property.Main && pulp test --test-path test/src"
+    "test-fast": "pulp test --test-path test/src --main Test.Fast",
```

Comment:
Yeah thats reasonable lets leave them named fast. : ). I think the short names thing is interesting though. Maybe we can do that in the future?